basic/0.entries/config.nix

```diff
@@ -14,7 +14,7 @@
 
    # can be any github ref
    # branch, tag, commit, etc.
-   ref = "master";
+   ref = "pr/bump-holo-nixpkgs-add-hc";
```

Comment:
if we decide to merge this PR we need to ensure a pinned revision or at least a stable branch

Reply:
Hi steveeJ,

Thanks for the pull request.
I tried it out locally. The nix script work well.
I noticed that I need to add a dna.yaml file, but more importantly, tryorama is not you up to date. As far as I can tell.
Is there a way to get a version for tryorama that can handle the new dna format?

This is the error I get:

18:38:18 [tryorama] error: Test error: {
  type: 'error',
  data: {
    type: 'internal_error',
    data: 'DnaError: Deserialize("missing field `dna`")'
  }
}
not ok 1 Test threw an exception. See output for details.